Tests/Shared.Specs/StringAssertionSpecs.cs

```diff
@@ -2043,6 +2043,138 @@ public void When_the_actual_string_equivalent_to_the_expected_but_with_trailing_
 
         #endregion
 
+        #region Not Be Equivalent To
+
+        [Fact]
+        public void When_strings_are_the_same_while_ignoring_case_it_should_throw()
+        {
+            //-----------------------------------------------------------------------------------------------------------
+            // Arrange
+            //-----------------------------------------------------------------------------------------------------------
+            string actual = "ABC";
+            string unexpected = "abc";
+
+            //-----------------------------------------------------------------------------------------------------------
+            // Act
+            //-----------------------------------------------------------------------------------------------------------
+            Action action = () => actual.Should().NotBeEquivalentTo(unexpected);
+
+            //-----------------------------------------------------------------------------------------------------------
+            // Assert
+            //-----------------------------------------------------------------------------------------------------------
+            action.Should().Throw<XunitException>()
```

Comment:
At least one test that call `Throw<XunitException>()` needs to verify that `because` and `becauseArgs` are included in the failure message.